webapp/channels/Makefile

```diff
@@ -1,26 +0,0 @@
-BUILD_SERVER_DIR = ../mattermost-server
-
-export NODE_OPTIONS=--max-old-space-size=4096
-
-.PHONY: i18n-extract
-i18n-extract: ## Extract strings for translation from the source code
-	npm run i18n-extract
-
-.PHONY: emojis
-emojis: ## Creates emoji JSON, JSX and Go files and extracts emoji images from the system font
-	SERVER_DIR=$(BUILD_SERVER_DIR) npm run make-emojis
-	@if [ -e $(BUILD_SERVER_DIR)/model/emoji_data.go ]; then \
-		gofmt -w $(BUILD_SERVER_DIR)/model/emoji_data.go; \
-	fi
```

Comment:
i went ahead and deleted the MakeFile for channels, as I didn't see the need for it because
- we are not passing the server dir as args in make-emojis
- update dependencies is something we do internally
- i18n-extract has npm command anyways

Reply:
yes added the gofmt in exec

Reply:
did go generate instead